src/helpdesk_app/views.py

```diff
@@ -29,6 +29,37 @@
 ##############################
 
 
+def thumbsUpClicked(request):
```

Comment:
Nit: the existing view functions use an underscore naming convention, could you do the same?